packages/block-editor/src/components/inserter/search-items.js

```diff
@@ -71,19 +71,6 @@ export const searchBlockItems = (
 			find( categories, { slug: item.category } )?.title,
 		getCollection: ( item ) =>
 			collections[ item.name.split( '/' )[ 0 ] ]?.title,
-		getVariations: ( { variations = [] } ) =>
```

Comment:
No longer referenced in the code as of https://github.com/WordPress/gutenberg/pull/25182.